docs/architecture.md

```diff
@@ -307,3 +290,62 @@ for more information.
 [Antrea for AKS
 Engine](https://github.com/Azure/aks-engine/blob/master/docs/topics/features.md#feat-antrea)
 and [Antrea EKS support](eks-installation.md) work in `NetworkPolicyOnly` mode.
+
+## Features
+
+### Antrea Network Policy
+Besides Kubernetes NetworkPolicy, Antrea supports two extra types of
+Network Policies available as CRDs - Antrea Namespaced NetworkPolicy and
+ClusterNetworkPolicy. The former is scoped to a specific Namespace, while the
+latter is scoped to the whole cluster. These two types of Network Policies
+extended Kubernetes NetworkPolicy with advanced features including: policy
+priority, tiering, deny action, external entity, and policy statistics. For more
+information about Antrea network policies, refer to the [Antrea Network Policy document](antrea-network-policy.md).
+
+The same as Kubernetes NetworkPolicies, Antrea Controller computes Antrea
```

Comment:
I don't think something can be computed to something else. Maybe you mean transform or convert?

Reply:
Changed to "transforms".